File: app/src/main/java/br/com/brqtest/database/dao/EnderecoDao.java
package br.com.brqtest.database.dao;

import br.com.brqtest.model.Endereco;
import com.j256.ormlite.dao.BaseDaoImpl;
import com.j256.ormlite.support.ConnectionSource;

import java.sql.SQLException;

public class EnderecoDao extends BaseDaoImpl<Endereco, Integer> {
    public EnderecoDao(ConnectionSource connectionSource) throws SQLException {
        super(Endereco.class);
        setConnectionSource(connectionSource);
        initialize();
    }
}

File: app/src/main/java/br/com/brqtest/databasekotlin/dao/EnderecoDao.kt
package br.com.brqtest.databasekotlin.dao

import br.com.brqtest.modelkotlin.Endereco
import com.j256.ormlite.dao.BaseDaoImpl
import com.j256.ormlite.support.ConnectionSource
import java.sql.SQLException


class EnderecoDao @Throws(SQLException::class)
constructor(connectionSource: ConnectionSource) : BaseDaoImpl<Endereco, Int>(Endereco::class.java) {
    init {
        setConnectionSource(connectionSource)
        initialize()
    }
}
